beta/docker/README.md

```diff
@@ -0,0 +1,231 @@
+
+5. Clone this repository and switch to this directory:
+
+    ```sh
+    git clone https://github.com/1Password/scim-examples.git
+    cd ./scim-examples/beta/docker
+    ```
+
+6. Save your `scimsession` file to the working directory.
+
+   > **Note**
+   >
+   > If you are deploying SCIM bridge directly on the Linux server, you will need to transfer the `scimsession` file
+   > from your local machine that is signed in to your 1Password account, for example:
+   >
+   > ```sh
+   > scp ./scimsession user@host:scim-examples/beta/docker/scimsession
+   > ```
+
+### Configure 1Password SCIM bridge to connect to Google Workspace
+
+If integrating 1Password with Google Workspace, additional cofiguration is required. See [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client) for instructions to create the service account, key, and API client.
+
+1. Copy the Google Workspace service account key to the working directory.
+
+   > **Note**
+   >
+   > If you are deploying SCIM bridge directly on the Linux server, you will need to transfer the Workspace credentials file
+   > from your local machine that is signed in to Google (or your 1Password account if you saved the credentials there), for example:
+   >
+   > ```sh
+   >  scp ./workspace-credentials.json user@host:scim-examples/beta/docker/workspace-credentials.json
+   > ```
+
+2. Copy the [settings file template](/beta/workspace-settings.json) from this repository:
+
+    ```sh
+    cp ../workspace-settings.json ./workspace-settings.json
+    ```
+
+    Edit the file. Replace the values for each key:
+
+    - `actor`: the email address for the administrator that the service account is acting on behalf of
+    - `bridgeAddress`: the URL for your SCIM bridge based on the DNS record created in [Get started](#get-started)
+
+    ```json
+    {
+        "actor":"admin@example.com",
+        "bridgeAddress":"https://scim.example.com"
+    }
+    ```
+
+    Save the file.
+
+## Deploy 1Password SCIM bridge
+
+After the DNS record has propagated, set the domain name for your SCIM bridge inline, use the template to output a canonical configuration that will create a Docker secret and deploy SCIM bridge, then create the stack from this configuration. You SCIM bridge should automatically acquire and manage a TLS certificate from Let's Encrypt on your behlaf using the supplied domain name.
+
+*Example command:*
+
+```sh
+OP_TLS_DOMAIN=scim.example.com docker stack config --compose-file ./compose.template.yaml |
+    docker stack deploy --compose-file - op-scim-bridge
+```
+
+Replace `scim.example.com` with the fully qualified domain name that points to your Linux server, then run the command to deploy your SCIM bridge.
+
+### Connect your SCIM bridge to Google Workspace
```

Comment:
The organization of the docs here may be a source of some confusion for Workspace customers. 

It's conceivable (I'd wager it's _likely_) that a Workspace customer will have already followed the directions at line 112.

This is and admittedly tricky one to navigate from a documentation standpoint, but I think we just need to do something to foreshadow to a Workspace customer, around line 112, that they should NOT run the command at line 112. 

This could either be a specific call-out like "if you are not using Google Workspace...."

Or it could be a matter of putting the two commands visually close to each other so Workspace and non-Workspace customers can easily see that there are two paths and pick the appropriate one. This could be perhaps putting _both_ behind disclosure arrows with titles that they can easily see the two paths and expand the relevant one.

Reply:
I tested and planned for this. When you run this command, you should get a new or updated stack for a SCIM bridge that is connected to Google Workspace. If you go through the general instructions on line 112 first, you get a SCIM bridge that's not connected to Google Workspace first until you run this one.